analytics/tests/test_counts.py

```diff
@@ -167,7 +167,7 @@ def assertTableState(self, table: Type[BaseCount], arg_keys: List[str],
 class TestProcessCountStat(AnalyticsTestCase):
     def make_dummy_count_stat(self, property: str) -> CountStat:
         query = """INSERT INTO analytics_realmcount (realm_id, value, property, end_time)
-                   VALUES (%s, 1, '%s', %%%%(time_end)s)""" % (self.default_realm.id, property)
+                   VALUES ({}, 1, '{}', %%(time_end)s)""".format(self.default_realm.id, property)
```

Comment:
Hmm, how do you feel about this type of conversion?  We mostly only have it in analytics code, and I suppose it's an improvement in that it gets rid of the double-escaping of `%s`?

Reply:
We should never be constructing SQL with any kind of string formatting operations. Apparently we still have such constructions that evaded #14852, and we should do something about it. Whether we do it before or after this PR doesn’t much matter.

Reply:
Let’s go for “before”. I did a hopefully more comprehensive sweep for computed query strings in #15270. Then I regenerated on this branch top of that and rebased it back, to avoid conflicting with it.